features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/metric/UploadBlockerMetric.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.rum.internal.metric
+
+internal enum class UploadBlockerMetric(val key: String) {
+    METRICS_KEY("[Mobile Metric] Batch Blocked"),
```

Comment:
I have a feeling that in this enum it is actually a mix of values and keys, am I wrong?